.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,89 @@
+name: Bug report
+description: Report a reproducible bug to help us improve
+title: "Bug: TITLE"
+labels: ["bug", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for submitting a bug report. Before you start, make sure that [the bug hasn't been reported already](https://github.com/awslabs/aws-lambda-powertools-typescript/issues).
+        
+        Please add as much information as possible to help us reproduce, and remove any potential sensitive data.
+  - type: textarea
+    id: expected_behaviour
+    attributes:
+      label: Expected Behaviour
+      description: Please share details on the behaviour you expected
+    validations:
+      required: true
+  - type: textarea
+    id: current_behaviour
+    attributes:
+      label: Current Behaviour
+      description: Please share details on the current issue
+    validations:
+      required: true
+  - type: textarea
+    id: code_snippet
+    attributes:
+      label: Code snippet
+      description: Please share a code snippet to help us reproduce the issue
+      render: JavaScript
+    validations:
+      required: true
+  - type: textarea
+    id: solution
+    attributes:
+      label: Possible Solution
+      description: If known, please suggest a potential resolution
+    validations:
+      required: false
+  - type: textarea
+    id: steps
+    attributes:
+      label: Steps to Reproduce
+      description: Please share how we might be able to reproduce this issue
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: AWS Lambda Powertools for TypeScript version
+      placeholder: "latest, 1.3.0"
+      value: latest
+    validations:
+      required: true
+  - type: dropdown
+    id: runtime
+    attributes:
+      label: AWS Lambda function runtime
+      options:
+        - 12.x
+        - 14.x
+        - 16.x
+    validations:
+      required: true
+  - type: dropdown
+    id: packaging
+    attributes:
+      label: Packaging format used
+      options:
+        - Lambda Layers
+        - Npm
+      multiple: true
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Debugging logs
+      description: If available, please share [debugging logs](https://awslabs.github.io/aws-lambda-powertools-python/#debug-mode)
+      render: Shell
+    validations:
+      required: false
+  - type: markdown
+    attributes:
+      value: |
+        ---
+
+        **Disclaimer**: We value your time and bandwidth. As such, any pull requests created on non-triaged issues might not be successful.
```

This sounds conflicting to me. What about:

```suggestion
        **Disclaimer**: After creating an issue, please wait until it is triaged and confirmed by a maintainer before implementing it. This will reduce amount of rework and the chance that a pull request gets rejected.
```